antha/AnthaStandardLibrary/Packages/download/urltofile.go

```
@@ -0,0 +1,38 @@
+// package for downloading files
+package download
+
+import (
+	"fmt"
+	"io"
+	"net/http"
+	"os"
+)
+
+// get a file from a url link, the file will be given the specified filename
+func UrlToFile(url string, filename string) (f *os.File, err error) {
```

so it should just return an error?